Variables/Undefined variable/task.md

```diff
@@ -3,11 +3,11 @@
 Variable names may only contain Latin letters, digits, and/or the underscore 
 character, and they cannot start with a digit. They also cannot be any of the 
 reserved <a href="https://docs.python.org/3/reference/lexical_analysis.html#keywords">keywords</a>.
+
 ### Task
-Check what happens if you use a variable which is not defined yet. Change and run the code in the editor – try to print out an undefined name.  
+Check what happens if you use a variable which is not defined yet. Change and run the code in the editor – try to print out an **undefined** name (i.e. type the name of an undefined variable inside the brackets of the `print` statement).  
```

Comment:
Check what happens if you use a variable that is not yet defined. Modify and run the code in the editor – try to print out an **undefined** name (i.e., type the name of an undefined variable inside the brackets of the `print` statement).